src/Page/Header.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC } from 'react';
+import { ThemeProvider } from '@emotion/react';
+import { StyledHeader } from './pageStyles';
+import theme from 'src/styles/theme';
+
+ /**
+ * Header Component
+ * 
+ * The Header component is used to render the header.
+ */
+const Header: FC = props =>
```

Comment:
The indentation in this file seems different than other component files. Can you update to match?